src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card--position_bottom:not(:nth-last-child(-n + 4)) {
+  margin-bottom: 46px;
+}
+
+.card__picture {
+  margin: 32px auto 40px;
+  height: 134px;
+}
+
+.card__img {
+  height: 100%;
+  width: auto;
+  object-position: center;
+}
+
+.card__info {
+  margin: 0 16px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 1.5em;
+  margin-top: 0;
+  margin-bottom: 4px;
+  color: $color-card-main;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
```

Comment:
` font-weight` by default is 400. You can not set it. On other places too

Reply:
@IvanFesenko I have 500 by default (see general.scss)